File: Curso_logica_de_programacao/Exercicios_logica_em_C/04-vetores/Soma_vetor/soma_vetor.c
#include <stdio.h>

int main()
{

    int n;
    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    double vet[n];

    for (int i = 0; i < n; i++)
    {
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }

    double soma = 0;
    for (int i = 0; i < n; i++)
    {
        soma = soma + vet[i];
    }
    double media = soma / n;

    printf("Valores = ");
    for (int i = 0; i < n; i++)
    {
        printf("%.2lf ", vet[i]);
    }

    printf("\nSoma = %.2lf", soma);
    printf("\nMedia = %.2lf", media);

    return 0;
}

File: Algoritmos_e_Logica_de_Programacao/Exercicios_logica_em_C#/Soma_vetor/Soma_vetor.cs
using System.Globalization;

namespace Soma_vetor;

class Program
{
    static void Main(string[] args)
    {

        CultureInfo CI = CultureInfo.InvariantCulture;

        int n;

        Console.Write("Quantos numeros voce vai digitar? ");
        n = int.Parse(Console.ReadLine());

        double[] vet = new double[n];

        for (int i = 0; i < n; i++)
        {
            System.Console.Write("Digite um numero: ");
            vet[i] = double.Parse(Console.ReadLine(), CI);
        }

        double soma = 0, media = 0;
        System.Console.Write("Valores = "); 
        for (int i = 0; i < n; i++)
        {
            System.Console.Write(" " + vet[i].ToString("F2", CI));
            soma += vet[i];
        }

        System.Console.WriteLine();

        media = soma / n;
        System.Console.WriteLine("Soma = " + soma.ToString("F2", CI));
        System.Console.WriteLine("Media = " + media.ToString("F2", CI));
        

    }
}
